openlibrary/plugins/upstream/account.py

```diff
@@ -325,7 +325,9 @@ def POST(self):
         if error:
             return self.render_error(error, i)
 
-        expires = (i.remember and 3600 * 24 * 7) or ""
+        expires = i.remember and 3600 * 24 * 7
```

Comment:
Is `None` a valid value in a web cookie?

Reply:
I think expires must be an int (or `None`) I think the fact that we were defaulting to `""` may actually be breaking "remember me"

From:
https://webpy.org/cookbook/cookies

Docs say:
> expires (int) - Optionally, is the time in seconds until the browser should expire the cookie. Note: this must be an integer, not a string.